sunpy/net/cdaweb/test/test_cdaweb.py

```diff
@@ -5,17 +5,14 @@
 from sunpy.net.cdaweb.attrs import Dataset
 
 
-@pytest.mark.remote_data
+@pytest.mark.vcr()
```

Comment:
Why do we need the `()` on this, that's not a normal requirement for pytest marks?

Reply:
You can normally add them and they behave the same as if they weren't there.